azkaban-common/src/test/java/azkaban/user/XmlUserManagerTest.java

```diff
@@ -102,19 +107,27 @@ public void testDoNotExist() throws Exception {
     fail("XmlUserManager should throw an exception when the file doesn't exist");
   }
 
+  private Path getFilePath(final String testName) throws IOException {
+    final URL configURL = Resources.getResource("test-conf/azkaban-users-test1.xml");
+    final String origPathStr = configURL.getPath();
+
+    // Create a new directory and copy the file in it.
+    final Path workDir = temporaryFolder.newFolder().toPath();
```

Comment:
Same question: with this change do we need to explicitly delete this folder (in two places below)?